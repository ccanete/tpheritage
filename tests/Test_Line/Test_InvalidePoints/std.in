L test 
L test 2
L test 2 4
L test 2 4 4
EXIT