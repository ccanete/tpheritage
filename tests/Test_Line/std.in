L test 2 4 4 8
EXIT