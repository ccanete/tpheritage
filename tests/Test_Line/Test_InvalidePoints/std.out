ERR
# Invalide first point.
ERR
# Invalide first point.
ERR
# Invalide second point.
ERR
# Invalide second point.

# The program is being closed.